docs/sources/alerting/fundamentals/_index.md

```diff
@@ -56,14 +61,15 @@ Whether you’re just starting out or you're a more experienced user of Grafana
 
 The following diagram gives you an overview of Grafana Alerting and introduces you to some of the fundamental features that are the principles of how Grafana Alerting works.
 
-{{< figure src="/media/docs/alerting/how-alerting-works.png" max-width="750px" caption="How Alerting works" >}}
+<br/>
+
+{{< figure src="/media/docs/alerting/alerting-configure-notifications-v2.png" max-width="750px" alt="How Alerting works" caption="How Alerting works" >}}
 
 ## How it works at a glance
 
-- Grafana alerting periodically queries data sources and evaluates the condition defined in the alert rule
+- Grafana Alerting periodically queries data sources and evaluates the condition defined in the alert rule
 - If the condition is breached, an alert instance fires
-- Firing and resolved alert instances are routed to notification policies based on matching labels
-- Notifications are sent out to the contact points specified in the notification policy
+- Firing (and resolved) alert instances are sent for notifications, either directly to a contact point or through notification policies for more flexibility
```

Comment:
@brendamuir, see also here the usage of "for notifications" as well as the "either/or"

Reply:
great